DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -315,15 +318,10 @@
     <field name="dtrkpt_" transient="true" />
     <field name="dphi_" transient="true" />
     <field name="deta_" transient="true" />
-    <field name="dxydxy_" transient="true" />
-    <field name="dzdz_" transient="true" />
-    <field name="dxydz_" transient="true" />
-    <field name="dlambdadz_" transient="true" />
-    <field name="dphidxy_" transient="true" />
-    <field name="dphidphi_" transient="true" />
-    <field name="detadeta_" transient="true" />
-    <field name="dptdpt_" transient="true" />
     <field name="track_" transient="true" />
+    <field name="m_" transient="true" />
+    <!--- <field name="covariance_load_flag" transient="true"/> -->
+   <!--- <field name="covarianceParameterization_" transient="true" />-->
```

Comment:
if I'm not mistaken, ```covarianceParameterization_``` is a static member of the class. Does it need to be persisted (or is it skipped somehow by root anyways?)